TouristEquipmentRental/RentEquipmentView.cpp

```diff
@@ -0,0 +1,136 @@
+#include "RentEquipmentView.h"
+
+void RentEquipmentView::RenderEquipmentList() {
+
+    system("cls");
+
+    auto fileDatabase = new FileDatabase();
+    auto equipment = fileDatabase->GetEquimpent();
```

Comment:
Would be better to make FileDatabase or DatabaseOperator an instance member of UniOfView class, and then pass a list of equipment as a parameter to this method from within UnitOfView.
There is also a possibility to make FileDatabase or DatabaseOperator an instance member of all views and pass to each view the same reference to it.